tests/integration/schema/crdt_type_test.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2022 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package schema
+
+import (
+	"testing"
+
+	"github.com/sourcenetwork/immutable"
+
+	"github.com/sourcenetwork/defradb/client"
+	testUtils "github.com/sourcenetwork/defradb/tests/integration"
+)
+
+func TestSchemaCreate_PNCounterTypeWithInt_NoError(t *testing.T) {
```

Comment:
todo: please revisit the tests' names again. One should be able to read the name alone in order to understand what it does

Reply:
✅